src/components/Table3.vue

```diff
@@ -37,8 +37,8 @@
               </td>
               <td>
                 <div class="tag-cell mx-5">
```

Comment:
Para que las etiquetas tengan una separación entre ellas:

```suggestion
                <div class="tag-cell mx-5 flex flex-wrap gap-3">
```